flow.js

```diff
@@ -30,9 +30,9 @@ declare type Context = SSRContext | ExtendedKoaContext;
 
 declare type FusionPlugin<Deps, Service> = {
   deps?: Deps,
-  provides?: (Deps: $ObjMap<Deps, ExtractReturnType>) => Service,
+  provides?: (Deps: $ObjMap<Deps & {}, ExtractReturnType>) => Service,
```

Comment:
These should remain synchronized with `flow-typed/fusion-core.js`.  This duplication is unfortunate, but should only be necessary for `fusion-core` today.